src/components/Grid/Grid.scss

```diff
@@ -0,0 +1,59 @@
+@import "./../../base/base.scss";
+
+.n-grid-container {
+  width: 100%;
+  display: grid;
+  gap: 1.5rem;
+  overflow: hidden;
+}
+
+@media screen and (max-width: 768px) {
+  .n-grid-container {
+    gap: 1rem;
+    -ms-overflow-style: none; /* Internet Explorer 10+ */
+    scrollbar-width: none; /* Firefox */
+  }
+  .n-grid-container::-webkit-scrollbar {
+    display: none; /* Safari and Chrome */
+  }
+
+  .overflow {
+    overflow: scroll;
+    gap: 0.75rem;
+  }
+  .overflow > * {
+    grid-row: 1;
+  }
+}
+
+.n-grid-block-1 {
+  grid-template-columns: repeat(1, 1fr);
+}
+
+.n-grid-block-2 {
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.n-grid-block-3 {
+  grid-template-columns: repeat(3, 1fr);
+}
+
+.n-grid-block-4 {
+  grid-template-columns: repeat(4, 1fr);
+}
+
+.n-grid-block-6 {
+  grid-template-columns: repeat(6, 1fr);
+}
+
+// If we want 8 column or 10 colum or 12 we can just add a new class here and it will automatically get added
```

Comment:
This doesnt seem dynamic. We cannot release new versions for support of extra columns. Please move this logic to JS and pass it as a style prop to the division

Reply:
@ShahRohan27  Yes now the column is also generating on js part

Reply:
Then we should consider removing this classes, what will happen in case of 7 children ?

Reply:
@ShahRohan27  All the classes are removed